.travis.yml

```diff
@@ -21,10 +21,10 @@ script: |
 jobs:
   include:
     - stage: deploy
-      if: type = push AND branch = master
+      if: type = push AND branch = master AND fork = false
```

Comment:
Do you maintain your own fork of `graphql-js` and trying to use Travis for it?
If not can you please describe your use case?

Reply:
When I have a fork of the repo AND I have Travis myself, this script will be run too.
This will fail my own pipeline because of the failure of deploy.

Doing this PR, the pipeline won't fail anymore

Reply:
Can you please extract it into separate PR?

Reply:
https://github.com/graphql/graphql-js/pull/2530